src/Driver/SQLite/Schema/SQLiteColumn.php

```diff
@@ -113,6 +114,9 @@ class SQLiteColumn extends AbstractColumn
      */
     protected bool $primaryKey = false;
 
+    #[ColumnAttribute([])]
```

Comment:
Can't the `string` type have the `size` option in SQLite?

Reply:
@roxblnfk SQLite ignores the size, as stated in the documentation:
https://www.sqlite.org/datatype3.html

> Note that numeric arguments in parentheses that following the type name (ex: "VARCHAR(255)") are ignored by SQLite - SQLite does not impose any length restrictions (other than the large global [SQLITE_MAX_LENGTH](https://www.sqlite.org/limits.html#max_length) limit) on the length of strings, BLOBs or numeric values.

Therefore, specifying it makes no sense.